conversion/conversion.go

```diff
@@ -305,7 +305,26 @@ func dataFromDatabase(ctx context.Context, sourceProfile profiles.SourceProfile,
 			dfJobId := dfOutput.JobID
 			gcloudCmd := dfOutput.GCloudCmd
 			streamingCfg, _ := streamInfo["streamingCfg"].(streaming.StreamingCfg)
-			streaming.StoreGeneratedResources(conv, streamingCfg, dfJobId, gcloudCmd, targetProfile.Conn.Sp.Project, "")
+			monitoringResources := metrics.MonitoringMetricsResources{
+				ProjectId:            targetProfile.Conn.Sp.Project,
+				DataflowJobId:        dfOutput.JobID,
+				DatastreamId:         streamingCfg.DatastreamCfg.StreamId,
+				GcsBucketId:          streamingCfg.TmpDir,
+				PubsubSubscriptionId: streamingCfg.PubsubCfg.SubscriptionId,
+				SpannerInstanceId:    targetProfile.Conn.Sp.Instance,
+				SpannerDatabaseId:    targetProfile.Conn.Sp.Dbname,
+				ShardId:              "",
+			}
+			respDash, dashboardErr := metrics.CreateDataflowShardMonitoringDashboard(ctx, monitoringResources)
+			var dashboardName string
+			if dashboardErr != nil {
+				dashboardName = ""
+				logger.Log.Error(fmt.Sprintf("Creation of the monitoring dashboard failed, please create the dashboard manually\n error=%v\n", dashboardErr))
+			} else {
+				dashboardName = strings.Split(respDash.Name, "/")[3]
+				fmt.Printf("Monitoring Dashboard: %+v\n", dashboardName)
```

Comment:
Curious - what does `respDash` contain?

Reply:
Dashboard response. The result of dashboard create request

Reply:
Hah! I meant what does it contain that we have to `strings.Split` 🙂, I assumed that they would return some sort of ID only.

Reply:
Yes, they return it in form of projects/<project-name>/dashboards/<dashboard-name>